### ISSUE TITLE
ldid in PATH not found in linux/macOS

### ISSUE THREAD

**Issue by FreQRiDeR** [itsnebulalol/permasigner#28]
## What is the expected behavior?
use ldid in path
## What is the actual behavior?
not found
## What are the steps to reproduce this bug?
run main.py
## Please provide debug logs:
Use the `--debug` option while running the script, and paste the output **into the codeblocks**.

```
terramoda@terramoda-700T1C:~$ ldid
Link Identity Editor v2.1.5-procursus3

Usage: ldid [-Acputype:subtype] [-a] [-C[adhoc | enforcement | expires | hard |
            host | kill | library-validation | restrict | runtime]] [-D] [-d]
            [-Enum:file] [-e] [-H[sha1 | sha256]] [-h] [-Iname]
            [-Kkey.p12 [-Upassword]] [-M] [-P] [-Qrequirements.xml] [-q]
            [-r | -Sfile.xml | -s] [-u] [-arch arch_type] file ...
Options:
   -S[file.xml]  Pseudo-sign using the entitlements in file.xml
   -Kkey.p12     Sign using private key in key.p12
   -Upassword    Use password to unlock key.p12
   -M            Merge entitlements with any existing
   -h            Print CDHash of file

More information: 'man ldid'
terramoda@terramoda-700T1C:~$ cd permasigner
terramoda@terramoda-700T1C:~/permasigner$ python3 main.py --debug
IPA Permasigner | Version main-77ff19f
Program created by Nebula | Original scripts created by zhuowei | CoreTrust bypass by Linus Henze

[DEBUG] Checking if command ldid is in PATH...
[*] ldid not found, downloading.
[DEBUG] Downloading ldid on Linux x86_64.


```

## Version Information
- OS Version: ubuntu 22.04/MacOS 12.4
- Jailbreak Name and Version: na

**Comment by FreQRiDeR** [comment]
Same in macOS Monterey. For the record, it's not an issue for me to use ldid in permasigner, just a fyi, if you wish to address. :)

**Comment by itsnebulalol** [comment]
Fixed in 74598a3
Run `git pull` and try again.
